admin-dev/themes/new-theme/js/app/widgets/ps-number.vue

```diff
@@ -78,25 +78,30 @@
       },
     },
     methods: {
+      getValue(): number {
+        const value = Number.isNaN(this.value) ? 0 : Number.parseInt(<string> this.value, 10);
+
+        return Number.isNaN(value) ? 0 : value;
```

Comment:
I don't understand this. Is it necessary to double check the same thing ? Couldn't we not just return the expression instead of going through a const  ?

Reply:
Wouldn't the first check `Number.isNaN(this.value)` catch it already ?

Reply:
Exactly @jolelievre! In a dark case, we must have an ultimate isNaN to check and set value correctly.